pepper-apis/src/main/resources/db-changes/schema/DDP-5604-activity-can-delete-instances.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20210222-activity-can-delete-instances-column">
+        <addColumn tableName="study_activity">
+            <column name="can_delete_instances" type="boolean" defaultValueBoolean="false">
```

Comment:
1. The ticket suggests the name `canDelete` but I think `canDeleteInstances` is more unambiguous.
2. The default value of `false` here ensures that existing activity definitions will NOT allow deletion, which is what we want.